app/assets/javascripts/modules/locationsearch.js

```diff
@@ -37,6 +37,9 @@ $(document).ready(function() {
       $('.leaflet-container').toggleClass('active');
     });
 
+    // hide zoom control on load
+    $('.leaflet-control-zoom').toggle();
```

Comment:
Should the zoom control reappear if the user toggles on the map search from the home page?